### PR TITLE
Clean up audit tool

### PR DESCRIPTION
**Pull request by KyleMacPherson**
Remove the Content Performance Manager code from the Audit so that future devs are able to iterate the codebase for the Audit tool without the burden of maintaining the legacy code for the CPM.

### CODE REVIEW

File: spec/domain/content/importers/all_content_items_spec.rb [hunk 1]
```diff
@@ -1,21 +0,0 @@
-module Content
-  RSpec.describe Importers::AllContentItems do
```
Comment (pmanrubia) [inline spec/domain/content/importers/all_content_items_spec.rb]: This test is needed, as importers are still a think for the Audit Tool

File: spec/domain/content/importers/all_google_analytics_metrics_spec.rb [hunk 2]
```diff
@@ -1,31 +0,0 @@
-module Content
-  RSpec.describe Importers::AllGoogleAnalyticsMetrics do
```
Comment (pmanrubia) [inline spec/domain/content/importers/all_google_analytics_metrics_spec.rb]: This test is needed, as importers are still a think for the Audit Tool

File: spec/domain/content/importers/single_content_item_spec.rb [hunk 3]
```diff
@@ -1,125 +0,0 @@
-module Content
```
Comment (pmanrubia) [inline spec/domain/content/importers/single_content_item_spec.rb]: This test is needed, as importers are still a think for the Audit Tool

File: spec/domain/content/link_filter_spec.rb [hunk 4]
```diff
@@ -1,32 +0,0 @@
-RSpec.describe Content::LinkFilter do
```
Comment (pmanrubia) [inline spec/domain/content/link_filter_spec.rb]: This test is needed, as the query is used in the Audit Tool

File: spec/domain/content/query_spec.rb [hunk 5]
```diff
@@ -1,175 +0,0 @@
-RSpec.describe Content::Query do
```
Comment (pmanrubia) [inline spec/domain/content/query_spec.rb]: This test is needed, as the query is used in the Audit Tool

File: spec/domain/content/sort_spec.rb [hunk 6]
```diff
@@ -1,118 +0,0 @@
-RSpec.describe 'sorting' do
```
Comment (pmanrubia) [inline spec/domain/content/sort_spec.rb]: This test is needed, as this still applies for the  Audit Tool

File: spec/models/content/link_spec.rb [hunk 7]
```diff
@@ -1,11 +0,0 @@
-RSpec.describe Content::Link do
```
Comment (pmanrubia) [inline spec/models/content/link_spec.rb]: This test is needed, as the Link is used in the Audit Tool

File: app/domain/content/importers/single_content_item.rb [hunk 8]
```diff
@@ -1,60 +0,0 @@
-module Content
```
Comment (pmanrubia) [inline app/domain/content/importers/single_content_item.rb]: This is still used in the Audit Tool

File: app/jobs/content/import_item_job.rb [hunk 9]
```diff
@@ -1,9 +0,0 @@
-module Content
-  class ImportItemJob < ApplicationJob
```
Comment (pmanrubia) [inline app/jobs/content/import_item_job.rb]: This applies in the Audit Tool

File: spec/features/import_spec.rb [hunk 10]
```diff
@@ -1,14 +0,0 @@
-require "rails_helper"
```
Comment (pmanrubia) [inline spec/features/import_spec.rb]: This test is needed

File: spec/jobs/content/import_item_job_spec.rb [hunk 11]
```diff
@@ -1,8 +0,0 @@
-module Content
-  RSpec.describe ImportItemJob, type: :job do
```
Comment (pmanrubia) [inline spec/jobs/content/import_item_job_spec.rb]: This test is needed, as the query is used in the Audit Tool

**Comment by andrewgarner** [comment]
You'll need to change the following:

- config/database.yml
- docker-compose.yml
- config/cable.yml
- config/initializers/session_store.rb
- README.md
- lib/tasks/heroku.rake
- config/environments/production.rb
- doc/development_workflow.md

**Comment by KyleMacPherson** [comment]
@pmanrubia I have pushed up the changes. Importing all items seems to be working without any problems locally.

**Comment by andrewgarner** [comment]
I'm going to rebase this to reword a few of the commit messages to fix some spelling and grammar mistakes.

**Pull request merged**